src/desktop/apps/partner2/stylesheets/contact.styl

```diff
@@ -17,6 +17,9 @@
   .partner2-locations
     .location
       margin-bottom 70px
+    div.location:last-child {
+      margin-bottom 20px
+      }
```

Comment:
#minor. We don't need curly braces in stylus, so we can safely remove them, and to be consistent with existing `location` class, maybe we can allow it on other tags than `div`:

```stylus
.location:last-child
  margin-bottom 20px
```